release notes/v0.53.0.md

```diff
@@ -0,0 +1,173 @@
+```
+
+or replace
+
+```javascript
+import { sleep } from "k6";
+```
+ with the CommonJS:
+
+```javascript
+const sleep = require("k6").sleep;
+```
+
+</details>
+#### Imported identifier that can't be resolved are now errors
+
+Previously to this if you were using the ESM syntax but imported identifier `foo` but the exporting file didn't export it - there will be no error.
+
+bar.js:
+```javascript
+export const notfoo = 5;
+```
+main.js
+
+```javascript
+import { foo } from "./bar.js"
+export default function () {
+    foo.bar(); // throws exception here
+}
+```
+It would not error out, but on accessing it - there will be an exception as it will be `undefined`.
+
+With native ESM support that is an error as defined by the specification and will error out sooner.
+
+This arguably is an improvement to UX/DX, but we have reports that there are users who have imports like this, but do not use them. So they wouldn't be getting exceptions, but they will now get errors.
+
+The solution in this case is to stop importing the not exported identifiers.
+
+### No more "magic" URL resolution
+
+For a long time k6 has supported special/magic URLs that aren't really that.
+
+Those were URLs without a scheme, that :
+1. started with `github.com` that if pasted in a browser will not open you the file. Their appeal was that you can more easily write them by hand if you know the path within a github repo.
+2. started with `cdnjs.com` that if pasted in a browser will open an web page with all the versions of the library. The appeal here is that you will get the latest version.
+
+Both of them had problems though.
+
+The github ones seemed to have never be used by users, likely because you need to guess what the path should look like, and you can always just go get a real URL to the raw file.
+
+While the cdnjs ones have some more usage, they are both a lot more complicated to support, as they require multiple requests to figure out what needs to be loaded. They also change over time. In addition the only known use at the moment is based on a very old example from an issue and it is even pointing to concrete, old version, of a library.
+
+Given that this can be done with a normal URL, we have decided to drop support for this and have warned users for the last couple of versions.
+
+### Deprecated `k6/experimental/tracing` in favor of a jslib
+
+`k6/experimental/tracing` is arguably not very well named, and there is good chances we would like to use the name for actual trace and span support within k6 in the future.
+
+On top of that it can now be fully supported in js code, which is why [http-instrumentation-tempo
+](https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo/) was created.
+
+The jslib is a drop in replacement so all you need to do is replace `k6/experimental/tracing` with `https://jslib.k6.io/http-instrumentation-tempo/1.0.0/index.js`.
+
+The module is planned to be removed in v0.55.0, planned for November 11th, 2024.
```

Comment:
I'm curious if connecting with the PR creates a connection in the task, which could be useful for future references 

```suggestion
The module is [planned to be removed in v0.55.0](https://github.com/grafana/k6/pull/3855), planned for November 11th, 2024.
```